src/enclave-agent/Cargo.toml

```diff
@@ -20,14 +20,15 @@ ctrlc = { version = "3.0", features = ["termination"] }
 tokio = { version = "1.14.0", features = ["full"] }
 async-trait = "0.1.42"
 
-image-rs = { git = "https://github.com/confidential-containers/image-rs", tag = "v0.2.0", features = ["occlum_feature"], default-features = false }
+image-rs = { git = "https://github.com/confidential-containers/image-rs", "rev" = "78a5114", features = ["occlum_feature", "cosign"], default-features = false }
```

Comment:
is it needed in this PR?

Reply:
> Another flag is the switch for `auth`. related link for `ImageConfig` https://github.com/confidential-containers/image-rs/blob/main/src/config.rs#L29. I'm afraid we also need to add one flag for that?

The `auth` mentioned by @Xynnn007 is introduced after `tag = "v0.2.0"`.

Reply:
these are not related to the fix in this PR and in fact now it's blocking getting this PR merged. I'd suggest we drop `image-rs` updates in this PR so we get it merged.

Reply:
Agreed. We can open an issue to trace registry authentication feature and impl in another PR.

Reply:
Sure, I'll drop the `auth` this PR.